app/helpers/settings_helper.rb

```diff
@@ -33,4 +33,8 @@ def translate_full_name(setting)
     fullname = setting.full_name.nil? ? setting.name : _(setting.full_name)
     trunc_with_tooltip(fullname, 32, setting.name, false)
   end
+
+  def render_if_partial_exists(path, setting)
```

Comment:
This is quite similar to `provider_partial_exist?`, and given the generic name I think it should move into one of the core layout helpers, or be settings-specific and have a settings-specific name prefix.